src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
```

Comment:
I noticed your entire specific code is in `main()`. Perhaps you can extract into functions and abstract the logic?